nextflow_schema.json

```diff
@@ -71,8 +66,7 @@
                     "pattern": "\\.fn?a(sta)?(\\.gz)?$",
                     "fa_icon": "fas fa-font",
                     "description": "Path to FASTA genome file.",
-                    "help_text": "This parameter is *mandatory* if `--genome` is not specified. If you don't have a BWA index available this will be generated for you automatically. Combine with `--save_reference` to save BWA index for future runs.",
-                    "fa_icon": "far fa-file-code"
```

Comment:
We don't want an icon here?

Reply:
There are several deleted icons, so I guess that this is intended, is it?

Reply:
Yeah, sorry, it is really difficult to update JSON files via the template sync so I done some of this when I merged the conflicts in https://github.com/nf-core/viralrecon/commit/b9c2a880203d9e5edb43cc80dd02074eda910975 and then the tests started failing because the sync messed the file up so I just copied across the schema before I merged in the conflicts that is why you see this diff 😏 I just loaded the Schema via `nf-core schema build` and all of the icons are there.